src/plugins/saved_search/server/saved_objects/search.ts

```diff
@@ -43,31 +44,33 @@ export function getSavedSearchObjectType(
         hideAggregatedPreview: { type: 'boolean', index: false, doc_values: false },
         hits: { type: 'integer', index: false, doc_values: false },
         kibanaSavedObjectMeta: {
+          dynamic: false,
           properties: {
+            // may need code changes
```

Comment:
I'm not really sure what this means -- what code changes may we need?

Reply:
If your code references any of the fields that could potentially be removed from the mappings, you might need to refactor. So, for example, if you're using `sort` from `savedObjectsAttributes` directly, you'd need to refactor to use `sort` from `_source`.